src/gui/embed.cpp

```diff
@@ -52,6 +52,15 @@ QPixmap getIconPixmap(const QString& pixmapName, int width, int height )
 	}
 	QImageReader reader(QString("artwork:%1").arg(pixmapName));
 
+	// xpm strings are not loaded from file, you must use addXpmToCache()
+	if (pixmapName.startsWith("xpm:"))
```

Comment:
I think it'd be better if this check were before creating a `QImageReader` instance.